petals-esb-deployer-impl/src/test/java/org/ow2/petals/deployer/utils/ModelDeployerImplTest.java

```diff
@@ -47,4 +49,17 @@ public void deployModel() throws Exception {
 
         verify(runtimeModelDeployerMock).deployRuntimeModel(any(RuntimeModel.class));
     }
+
+    @Test
+    public void deployModelWithMavenUrl() throws Exception {
+        System.setProperty("java.protocol.handler.pkgs", "org.ops4j.pax.url");
+
+        try {
+            modelDeployer.deployModel(ModelUtils.generateTestModelMavenUrl());
+        } catch (ModelDeploymentException e) {
+            assertFalse(e.getMessage().contains("unknown protocol: mvn"));
```

Comment:
This assertion is environment dependant. It fails is the unit test runs on a french env. Perhaps can you check the type of cause instead of the exception message ?